tests/unit/dataactvalidator/test_a16_appropriations.py

```diff
@@ -1,13 +1,31 @@
-from dataactcore.models.stagingModels import Appropriation
-from tests.unit.dataactvalidator.utils import error_rows, number_of_errors
+from tests.unit.dataactcore.factories.staging import AppropriationFactory
+from tests.unit.dataactcore.factories.job import SubmissionFactory
+from tests.unit.dataactvalidator.utils import error_rows, number_of_errors, insert_submission
 
 
 _FILE = 'a16_appropriations'
 
 
 def test_success(database):
-    award = Appropriation(job_id=1, row_number=1)
-    assert error_rows(_FILE, database, models=[award]) == []
+    sub_new = SubmissionFactory()
+    sub_prev_published = SubmissionFactory()
```

Comment:
The submission factory sets publishable to False and publish_status_id is left null.  I'm setting it in each test case that needs it.  This line currently sets the status to published, perhaps this comment was on an older commit?